### ISSUE TITLE
Error on setting Elevation 

### ISSUE THREAD

**Issue by TimorSol** [caewok/fvtt-elevation-ruler#84]
Foundry 12.327
System A5e 0.19.7

When using elevation ruler's [ and ] to change height we get a error calling for elevation to be a finite number. 
![obraz](https://github.com/caewok/fvtt-elevation-ruler/assets/146195121/cc1c9966-f5a6-44b8-af1e-cb93ab5cc90c)
![obraz](https://github.com/caewok/fvtt-elevation-ruler/assets/146195121/01f82c26-dd8e-4e0c-8394-cc0cc0cb47db)
![2024-06-13 09-55-57](https://github.com/caewok/fvtt-elevation-ruler/assets/146195121/f2d73fb3-f6a6-402a-ad01-d8a4f4378377)

This might be due to the system or due to the module, but it does provoke a module error so I'm posting.

**Comment by 7H3LaughingMan** [comment]
Getting a similiar issue with the PF2e system.

![image](https://github.com/caewok/fvtt-elevation-ruler/assets/117857081/46a876bc-4751-4d83-afdf-39d0caffe26c)

I tracked it down to this section of code in the module, setting a break point revealed that it's trying to update the elevation to `NaN`.

```
// Update elevation after the token move.
  if ( segment.ray.A.z !== segment.ray.B.z ) {
    const multiple = Settings.get(Settings.KEYS.TOKEN_RULER.ROUND_TO_MULTIPLE) ?? 1;
    const elevation = CONFIG.GeometryLib.utils.pixelsToGridUnits(segment.ray.B.z).toNearest(multiple);
    await token.document.update({ elevation });
  }
```

**Comment by 7H3LaughingMan** [comment]
Setting **Round Distance to Multiple** to 1 in the settings for Elevation Ruler seems to fix the issue. So it seems like the problem occurs when the rounding is disabled.

**Comment by Mystler** [comment]
I believe it's a division by 0 when 0 is passed into toNearest(). A simple workaround should be to only call that when multiple !== 0.

**Comment by caewok** [comment]
Should be addressed in v0.9.5.
